astroquery/template_module/core.py

```diff
@@ -36,15 +36,27 @@
 @async_to_sync
 class TemplateClass(BaseQuery):
 
+    def __init__(self, url='', timeout=None):
+        self.url = url          # A falsy default that cannot be mistaken for a valid value.
+        self.timeout = timeout  # Use `None` as default if the falsy value could be valid.
```

Comment:
I'm not sure our users will all be familiar with the term "falsy" - maybe clarify, e.g., `("falsy" means in this case bool(url) is False)`

Reply:
The target audience for these comments are developers, but I added an explanation nonetheless.

Reply:
our developers are not necessarily experienced, and in particular, not necessarily experienced in python (we have many people coming from Java backgrounds, for example)

Reply:
But then "falsy" is not a python specific term :) Anyway, I don't think it matters either way. Also, in my experience, many new modules are taking another similar module as a template rather than the "template" one, so we may even want to rethink the purpose of the template.